packages/core/test/types.test.ts

```diff
@@ -451,16 +451,16 @@ describe('events', () => {
 });
 
 describe('interpreter', () => {
+  // TODO: figure out why this doesn't pass
```

Comment:
Since CI is green and this test is not skipped I assume that this has already been fixed:
```suggestion
```